### ISSUE TITLE
`institute` YAML tag breaks latex output

### ISSUE THREAD

**Issue by IvoBCD** [jgm/pandoc#3029]
An input file containing the `institute` YAML tag causes an error with the default latex template:

```
---
institute: AnyInstitute
---

Blah
```

```
$ pandoc pandoc-bug.txt -t latex -o bug.pdf
! Undefined control sequence.
l.49 \institute

pandoc: Error producing PDF
```

(tested with pandoc v1.17.2)

**Comment by jgm** [comment]
See jgm/pandoc-templates#204

**Comment by jgm** [comment]
This should now be fixed with d9880fe3408baecd8ff7ae7a74291feea6f984e7
